python/src/WeibullMax_doc.i.in

```diff
@@ -1,6 +1,9 @@
 %feature("docstring") OT::WeibullMax
 "WeibullMax distribution.
 
+Available constructors:
+    WeibullMax(*beta=1.0, alpha=1.0, gamma=0.0*)
```

Comment:
idem: redundant with the parameter list below

Reply:
idem: it is mentionned only to get easily the default values....